keyboards/primekb/prime_o/keymaps/reasonsandreasons/keymap.c

```diff
@@ -0,0 +1,187 @@
+/* Copyright 2018 Jumail Mundekkat
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define L1BS LT(2, KC_BSPC)
+#define LSUP LSFT_T(KC_UP)
+#define CTLL LCTL_T(KC_LEFT)
+#define ECTL LCTL_T(KC_ESC)
+#define ALLM ALL_T(KC_MINS)
+#define SCLS LT(3, KC_SCLN)
+#define ______ KC_TRNS
```

Comment:
If you use 7 spaces instead of 6, you won't need to define this as including `QMK_KEYBOARD_H` already defines it.

Reply:
Should be updated! Thanks!